### ISSUE TITLE
Profiler does not accept to run tasks with options

### ISSUE THREAD

**Issue by HelainSchoonjans** [gradle/gradle-profiler#211]
I want to run the profile on a task with options for my target task:

    gradle-profiler --benchmark  "buildDocker --parallel -x test"

The command will fail with error:

    Caused by: org.gradle.internal.exceptions.LocationAwareException: Task 'buildDocker --parallel -x test' not found in root project 'MyProject'.

Another example:

    gradle-profiler --benchmark clean -x test

Gives the error:

    x is not a recognized option

**Comment by runningcode** [comment]
You can create a performance.scenario file if you would like to benchmark this scenario with the different flags. https://github.com/gradle/gradle-profiler#advanced-profiling-scenarios

**Comment by wolfs** [comment]
FWIW, I didn't manage to use Gradle profiler with a `-x ...` command line for the build, neither from the command line nor via a scenario file. @runningcode Do you have an example for a working scenario file which uses `-x <someTask>`?

**Comment by runningcode** [comment]
That's true. The `-x` doesn't work. Here's a related issue: https://github.com/gradle/gradle-profiler/issues/210

**Comment by erichaagdev** [comment]
I'm not sure if this is a recent enhancement, but you can supply `-x` from a scenarios file using `gradle-args`:

```
base-scenario {
  cleanup-tasks = ["clean"]
  tasks = ["build"]
  daemon = "cold"
  warm-ups = 1
  iterations = 1
  gradle-args = [
    "--no-build-cache",
    "-x", "dockerTest",
    "--dry-run"
  ]
}
```

In fact, you can use this option to supply any Gradle arguments. I'm unsure of how to do this from the command line, though. 

See: https://ge.solutions-team.gradle.com/s/nkhonssggeys2

**Comment by tresat** [comment]
Closing this as a duplicate of:
- https://github.com/gradle/gradle-profiler/issues/210

This needs the same fix: better documentation.
